build/scripts/site-config-version-update/main.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements a program that replicates supported version of development to production.
+package main
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/pelletier/go-toml"
```

Comment:
Shouldn't there be a go.mod in here? Given this dependency?

Reply:
The go.mod and go.sum files are now in place.